chainguard-source

```diff
@@ -159,8 +185,14 @@ handle_ref() {
 			tar xf "$target_filename" -C $(dirname "$target_filename")
 		;;
 		pkg:github/*)
-			info "Checking out [$url]"
-			git_checkout "$url"
+			# If we're not running in PRIVILEGED mode, then skip any pkg urls that end in .yaml
+			if [ "$PRIVILEGED" = "0" ] && echo "$url" | grep -q '\.yaml$'; then
```

Comment:
This excludes melange yaml files in wolfi-dev/os - was this intentional? that is a public repository so those can be retrieved without using authenticated access to github.

Reply:
This is a pickle indeed. We decided not to download any yaml files in unprivileged mode as to not confuse users thinking that 'all the yamls' are there, even though they're not (since most might not have been downloaded).
I did the same for apks but we need the apk *sometimes* because, for unknown reasons, the image SBOMs not always include all the source links for all the packages. Sometimes we need to fetch the apk and recursively look at its spdx file to get the actual source :|
I'm fine with either way actually, since none of the solutions are truly clean. We can always modify this in a next PR too!